pontoon/base/static/js/translate.js

```diff
@@ -5,45 +5,54 @@ var Pontoon = (function (my) {
     /*
      * UI helper methods
      */
-    getFilter: function() {
-      return $('#filter').data('current-filter') || 'all';
+    getFilter: function(type) {
+      return type ? $('#filter').data('current-filter')[type] : $('#filter').data('current-filter') || {
+        type: '',
+        author: '',
+        time: ''
+      };
     },
 
 
     /*
      * Set filter widget
      */
-    setFilter: function (type) {
-      type = type || 'all';
-      var node = $('#filter .menu [data-type="' + type + '"]'),
-          selectorType = type;
+    setFilter: function (filter) {
```

Comment:
I think we can rename this function into `setFilters` to be more accurate. Also, could you add a docstring about required structure of passed parameter?